databuilder/extractor/bamboohr_user_extractor.py

```diff
@@ -0,0 +1,65 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+
+from pyhocon import ConfigTree
```

Comment:
thanks for the contribution, could we move the module to `databuilder/extractor/user/boomboohr` ? this helps to add other user extractor later. Also eventually I plan to refactor and do the same dataset/table extractor. thanks.

Reply:
Thanks.  Will do.

Looks like I also have a unit test failure that I'll need to debug.